src/grid/Makefile

```diff
@@ -6,40 +6,40 @@ clean:
 	rm -fv *.o */*.o *.x ../offload/*.o
 
 CFLAGS := -fopenmp -g -O3 -march=native -Wall -Wextra
-NVFLAGS := -g -O3 -lineinfo -arch sm_70 -Wno-deprecated-gpu-targets -Xcompiler "$(CFLAGS)" -D__GRID_CUDA
+NVFLAGS := -g -O3 -lineinfo -arch sm_70 -Wno-deprecated-gpu-targets -Xcompiler "$(CFLAGS)" -D__OFFLOAD_CUDA
 LIBS := -lm -lblas
 
 ALL_HEADERS := $(shell find . -name "*.h") $(shell find ../offload/ -name "*.h")
 ALL_OBJECTS := ../offload/offload_buffer.o \
-               ../offload/offload_library.o \
-               grid_replay.o \
-               grid_task_list.o \
-               common/grid_library.o \
-               common/grid_basis_set.o \
-               common/grid_sphere_cache.o \
-               ref/grid_ref_task_list.o \
-               ref/grid_ref_collocate.o \
-               ref/grid_ref_integrate.o \
-               ref/grid_ref_prepare_pab.o \
-               cpu/grid_context_cpu.o \
-               cpu/coefficients.o \
-               cpu/grid_collocate_dgemm.o \
-               cpu/grid_integrate_dgemm.o \
-               cpu/non_orthorombic_corrections.o \
-               cpu/utils.o \
-               cpu/collocation_integration.o \
-               cpu/grid_context_cpu.o \
-               cpu/grid_prepare_pab_dgemm.o \
-               cpu/tensor_local.o
+															../offload/offload_library.o \
```

Comment:
This indentation is pretty wild. Unfortunately, there seems to be no tool for formatting Makefiles.

Reply:
emacs is the culprit. space / tab... I will fix it.

Reply:
Thanks! In the meantime I've committed #1930. So, in the future `make pretty` should take care of it.

Reply:
Thanks. I just saw your message. I need to do a make pretty again. Had issues with rebasing.